internal/server/server_test.go

```diff
@@ -308,7 +318,7 @@ func TestComposeImage(t *testing.T) {
 		}
 		response, body := tutils.PostResponseBody(t, "http://localhost:8086/api/image-builder/v1/compose", payload)
 		require.Equal(t, 400, response.StatusCode)
-		require.Contains(t, body, "Exactly one upload request should be included")
+		require.Contains(t, body, "Value is not nullable")
```

Comment:
Hmm I think it would make sense to expand the error message matching here a little to something less generic, the full error is 
```
{"errors":[{"detail":"request body has an error: doesn't match the schema: Error at \"/image_requests/0/upload_request/options\": Value is not nullable\nSchema:\n  {\n    \"anyOf\": [\n      {\n        \"$ref\": \"#/components/schemas/AWSUploadRequestOptions\"\n      },\n      {\n        \"$ref\": \"#/components/schemas/GCPUploadRequestOptions\"\n      },\n      {\n        \"$ref\": \"#/components/schemas/AzureUploadRequestOptions\"\n      }\n    ]\n  }\n\nValue:\n  null\n","title":"400"}]}
```

Like matching `image_requests/0/upload_request/options` to make sure it's the options which aren't nullable.

Reply:
Sounds good, I'll make changes, thanks!